frontend/hooks/useTeamIdParam.ts

```
@@ -226,20 +237,31 @@ const getDefaultTeam = ({
+          defaultTeam = preferredOrLowestIdFleet(realFleets);
         } else {
           defaultTeam = userTeams.find((t) => t.id === APP_CONTEXT_NO_TEAM_ID);
         }
       }
     }
 
-    return defaultTeam || userTeams.find((t) => t.id > APP_CONTEXT_NO_TEAM_ID);
+    if (!defaultTeam) {
+      const realFleets = userTeams.filter(
+        (t) => t.id > APP_CONTEXT_NO_TEAM_ID
+      );
```

ESLint

[error] Replace `⏎········(t)·=>·t.id·>·APP_CONTEXT_NO_TEAM_ID⏎······` with `(t)·=>·t.id·>·APP_CONTEXT_NO_TEAM_ID`

(prettier/prettier)